### ISSUE TITLE
`github/require-passive-events` false-positive when `passive: false` is passed

### ISSUE THREAD

**Issue by boris-petrov** [github/eslint-plugin-github#355]
```js
window.addEventListener('touchmove', touchMove, { passive: false });
```

This should not trigger a `github/require-passive-events` warning because I've explicitly passed `passive: false`.
